samples/apps/csv-data-chat-webapp-streamlit/requirements.txt

```diff
@@ -0,0 +1,4 @@
+semantic-kernel

```

Comment:
We try to work with Poetry as much as possible. If you do want to make this a standalone demo, could you create a Poetry project for this sample, please?

Also, is there anything else that a user/dev would need to know to get this running, for example, documented in a README.md file?